### PR TITLE
DOCS-166: update docs nav for dev-tools

### PR DESCRIPTION
**Pull request by kay-kim**
1. Update Docs top nav bar to have Developer Tools instead of Client Libraries to allow for SDK (also matches products page of company website)
Screen Shot 2020-09-15 at 4 58 48 PM

2. Add redirects since want DevTools instead of Client Libraries

3. Pin version of the theme we're using until we can change the template at a later date.

- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: packages/browser/docs/api/_templates/docs-navbar.html [hunk 1]
```diff
@@ -12,16 +12,18 @@
...
           </li>
-          <li class="nav-item"><a class="nav-link" href="#">Client Libraries </a>
+          <li class="nav-item"><a class="nav-link" href="#">Developer Tools</a>
                <ul class="nav-children">
                      <li class="nav-item nav-childitem">
-                        <a href="{{theme_clientlibjs_docs}}"> JavaScript</a>
+                        <a href="{{theme_clientlibjs_docs}}">JavaScript Client Libraries</a>
+                     </li>
+                     <li class="nav-item nav-childitem">
+                        <a href="{{theme_reactsdk_docs}}">JavaScript React SDK</a>
```
Comment (Vinnl) [inline packages/browser/docs/api/_templates/docs-navbar.html:23]: @kay-kim Given that https://github.com/inrupt/docs-client-lib-js/pull/9 says "Wait until next week", should this wait as well?

**Review by Vinnl — CHANGES_REQUESTED** [review]
Holding off until Sept 23rd to merge this.

**Comment by kay-kim** [comment]
Oops -- forgot about this pull.  Ping @NSeydoux @Vinnl

**Review by Vinnl — APPROVED** [review]
Whoops, I'd set a reminder for solid-client, but not this one apparently.

**Pull request merged**